main.yaml

```diff
@@ -275,6 +276,7 @@ x-tagGroups:
       - Organization Job Templates
       - Organization Job Template Locales
       - Comments
+      - Comment reactions
```

Comment:
IIRC some generated classes are named after these tags. Not sure whether it is affected by the case of the tags, but looking that all other tags are capitalized, I suggest renaming this to `Comment Reactions`, with both words capitalized.